### ISSUE TITLE
Unable to log in via GitHub in Atom UI.

### ISSUE THREAD

**Issue by trusktr** [Floobits/floobits-atom#120]
I can log into GitHub just fine in Chrome, but in the Floobits tab inside Atom GitHub says wrong username or password. The reason seems to be that when I paste my password from the clipboard, it pastes it twice.

For example, if my password is foobar, then it pastes "foobarfoobar". I verified this by pasting into the username field where I can see what is pasted. So, as workaround, I simply backspace half the characters.

**Comment by SamMorrowDrums** [comment]
This is ridiculous. In the words of John Oliver "IT IS 2017!!!!!!!!!!!"

"Not being able to sign-in to floobits because of a copy-paste bug, how is this still a thing?!"

**Comment by idan** [comment]
This is pretty frustrating. The workaround works, but I spent a while getting locked out of github for too many invalid signin attempts. With password managers generating 64 characters, the issue is completely obscured by the innumerable `••••••••••••••••`'s — no indication that it's double the amount of chars.

tl;dr grateful that I haven't been permablocked from Github for countless failed logins because of this plugin. 😞

**Comment by catherinedevlin** [comment]
Thank you SO MUCH for the workaround, which I never never would have figured out.

Confirming that the bug is still there in 0.32.10 on Atom 1.16.0 x64 on OS X...

**Comment by greelgorke** [comment]
this bug is still there... wow
